app/components/DataTable/components/ColumnPicker/index.tsx

```diff
@@ -77,7 +77,7 @@ const ColumnPicker = ({
                     onChange={(event) => handleChange(event, row?.getColId())}
                   />
                 )}
-                label={row.name}
+                label={row.name ? row.name : row.getColId()}
```

Comment:
I realize this was already called this, but renaming `row` to `col` would be a less confusing variable name. also rather than using name/colId here I would use [`col.getDisplayNameForColumn()` ](https://www.ag-grid.com/javascript-data-grid/column-api/#reference-Visibility%20and%20Display-getDisplayNameForColumn) as that will match what they see in the column header

Reply:
oh cool, yeah I'll try that method

Reply:
looks like current version of agGrid we have does not support `getDisplayNameForColumn`

Reply:
bummer